packages/pointer_interceptor/pointer_interceptor/CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
-## NEXT
+## 0.10.1+1
 
 * Updates support matrix in README to indicate that iOS 11 is no longer supported.
 * Clients on versions of Flutter that still support iOS 11 can continue to use this
   package with iOS 11, but will not receive any further updates to the iOS implementation.
+* Removes invalid `implements` tag in pubspec.
```

Comment:
Publishing for this change could be omitted, I think?

Reply:
Yea, I wasn't sure whether this affected clients or not. I did it just in case.